FitNesseRoot/FitNesse/UserGuide/WritingAcceptanceTests/SliM/ExceptionHandling/content.txt

```diff
@@ -8,6 +8,10 @@ If your exception indicates an error that cannot be recovered from, then you may
 
 By throwing an exception with "!-StopSuite-!" in the class name you can stop both current test and the suite execution. Tear down procedure for the running test and the suite will still be executed, but all the consequent tests in the suite will not run.
 
+!2 Ignoring Tests during execution
+If you want to ignore or skip certain parts of your test during execution you can do this by throwing an ignore exception. By throwing an exception with "!-IgnoreScriptTest-!" in the class name you can ignore and skip the instructions in the rest of that script table.
+If you want to ignore or skip all subsequent parts of the current batch during execution you can do this by throwing an ignore all exception. By throwing an exception with "!-IgnoreAllTests-!" in the class name you can ignore and skip all instructions in the current batch.
```

Comment:
What do you mean with 'current batch', the current test page, or suite, or ...?

Reply:
@fhoeben I used that term as it was used previously for the StopTest exception handling functionality. But I've reworded this line now in the last commit.